src/pages/home/RecentRecords.vue

```
@@ -261,16 +261,43 @@ const updateTopicHeaders = async () => {
   }
 };
 
-const toggleColumnExpanded = (index) => {
-  wwHeaders.value[index].expanded = !wwHeaders.value[index].expanded;
+const wwExpanded = ref(false);
+
+const toggleAllWwColumns = () => {
```

ESLint

[error] 'toggleAllWwColumns' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)